osu.Game/Rulesets/Mods/ModMetronome.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Audio.Track;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.UI;
+using osu.Game.Skinning;
+using osu.Game.Configuration;
+using osu.Framework.Bindables;
+using osu.Game.Audio;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModMetronome : Mod
+    {
+        public override string Name => "Metronome";
+        public override string Acronym => "MT";
+        public override IconUsage? Icon => OsuIcon.ModAuto; // temporary icon
```

Comment:
```suggestion
        public override IconUsage? Icon => FontAwesome.Solid.TachometerAlt;
```
probably closest you can get to an actual metronome